### PR TITLE
Use HTTPS for links to the docs

### PR DESCRIPTION
**Pull request by bocharsky-bw**
It was done in Symfony Demo https://github.com/symfony/symfony-demo/pull/550

### CODE REVIEW

File: app/Resources/views/default/index.html.twig [hunk 1]
```diff
@@ -19,7 +19,7 @@
             <div id="next">
                 <h2>What's next?</h2>
                 <p>
-                    <svg id="icon-book" version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" viewBox="-12.5 9 64 64" enable-background="new -12.5 9 64 64" xml:space="preserve">
+                    <svg id="icon-book" version="1.1" xmlns="https://www.w3.org/2000/svg" x="0px" y="0px" viewBox="-12.5 9 64 64" enable-background="new -12.5 9 64 64" xml:space="preserve">
```
Comment (stof) [inline app/Resources/views/default/index.html.twig]: that's wrong. The official XML namespace for SVG does not have `https` in it (the website uses HTTPS, but this does not impact the XML namespace)

File: phpunit.xml.dist [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <!-- https://phpunit.de/manual/current/en/appendixes.configuration.html -->
-<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.8/phpunit.xsd"
+<phpunit xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
```
Comment (stof) [inline phpunit.xml.dist]: that's wrong as well

File: app/Resources/views/default/index.html.twig (hunk 1 above)
Comment (bocharsky-bw) [reply app/Resources/views/default/index.html.twig]: Ah, I followed the link and was redirected to the `https`, that's why I decided to change it.

Thanks, I'll revert it!

**Comment by fabpot** [comment]
Thank you @bocharsky-bw.

**Pull request closed without merging**